components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/executor/PasswordProvisioningExecutor.java

```diff
@@ -161,10 +161,17 @@ private ExecutorResponse handleAskPasswordFlow(FlowExecutionContext context, cha
             context.getFlowUser().setUserId(userId);
             return new ExecutorResponse(STATUS_COMPLETE);
         } catch (UserStoreException | IdentityEventException | IdentityRecoveryException e) {
+            ExecutorResponse errorResponse =
+                    handleClientExceptionForActionFailure(new ExecutorResponse(), e);
```

Comment:
Why do we need to pass an empty ExecutorResponse() object to the handleClientExceptionForActionFailure() method ?